process/elasticproc/elasticProcessor.go

```diff
@@ -56,6 +56,7 @@ type ArgElasticProcessor struct {
 	DBClient           DatabaseClientHandler
 	LogsAndEventsProc  DBLogsAndEventsHandler
 	OperationsProc     OperationsHandler
+	InnerTxsHandler    InnerTxsHandler
```

Comment:
missing nil check

Reply:
the check for nil is the `check.go` file and it's done on line 82.